openupgrade_scripts/scripts/account/14.0.1.1/upgrade_analysis_work.txt

```diff
@@ -133,10 +133,10 @@ account      / account.group            / code_prefix (char)            : DEL
 account      / account.group            / code_prefix_end (char)        : NEW
 account      / account.group            / code_prefix_start (char)      : NEW
 # DONE: pre-migration: renamed code_prefix to code_prefix_start
-# post-migration: assigned same start values to end field
+# DONE: post-migration: assigned same start values to end field (why?)
```

Comment:
Maybe because in 13.0 there is no concept of an account group 'range', so this is the only valid value upon migration? And the upstream model takes much care to keep the values in sync if only one of both is set, in its onchange and write and create methods.